katello/katello-backup

```diff
@@ -7,14 +7,20 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
+@options[:features] = []
```

Comment:
I initialized it outside of the option parser so it won't be nil [here](https://git.io/vHMoW), it makes the logic a bit easier in that function instead of handling nil. Is there a specific concern?